### DISCUSSION TITLE
New model

### DISCUSSION THREAD

**Discussion by ilmaino1984** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#366] [General]
Hi, congratulations for the excellent work
some possibility of implementing support to the model  Xiaomi Robot Vacuum-Mop 2S (ijai.vacuum.v19)?
This is my actual situation

![Immagine 2022-11-14 183355](https://user-images.githubusercontent.com/58007629/201727957-c3b4be54-09eb-41fb-97b5-a8b30630908e.jpg)

Thanks
Regards

**Comment by PiotrMachowski** [comment]
#200
